docs/src/advanced-concepts/fitting-generic-targets.rst

```diff
@@ -1,11 +1,52 @@
-Preparing generic targets for reading by metatrain
-==================================================
+Fitting generic targets
+=======================
 
 Besides energy-like targets, the library also supports reading (and training on)
 more generic targets.
 
+Support for generic targets
+---------------------------
+
+Not all architectures can train on all types of target. Here you can find the
+capabilities of the architectures in metatrain.
+
+.. list-table:: Sample Table
+   :header-rows: 1
+
+   * - Target type
+     - Energy and its gradients
+     - Scalars
+     - Spherical tensors
+     - Cartesian tensors
+   * - SOAP-BPNN
+     - Energy, forces, stress/virial
+     - Yes
+     - Only with ``o3_lambda=1, o3_sigma=1``
+     - No
+   * - Alchemical Model
+     - Energy, forces, stress/virial
+     - No
```

Comment:
not really undestand here, these models should predict scalars right?

Reply:
They can predict scalars, but not arbitrary scalars... only energies. With the current design, the energy is a scalar with a single output that can have position and strain derivatives.